### PR TITLE
Feature/remove dist dir

### PR DESCRIPTION
**Pull request by day01**
In the PR:

*removed "dist" from git repository (there were a lot of unnecessary diffs when compiling) - js should be in npm package, doesn't in git

### CODE REVIEW

**Comment by day01** [comment]
@aitboudad, as you asked.

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ typings
 *.tgz
 .idea
 .tmp
+dist
+app/build
```
Comment (aitboudad) [inline .gitignore]: we can keep the `app/build` as it's used for demo

Comment (day01) [inline .gitignore]: Shouldn't we build the application before demo? 
I don't see demo on github as a static code. Are you sure it should be in a repository? I don't know a context, that why I am asking :)

Comment (aitboudad) [inline .gitignore]: it was supposed to work using http://rawgit.com, but not sure if it's still valid or not.

Comment (day01) [inline .gitignore]: This argument makes sense.

**Comment by allenhwkim** [comment]
I would like close this PR, removing dist and build directory and reverting back does not improve  this module, IMO.

**Pull request closed without merging**